dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.vcxproj

```diff
@@ -174,7 +174,7 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="..\..\packages\Microsoft.FrameworkUdk.$(Udk-Arch)fre.10.0.22431.1002-210816-1731.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.$(Udk-Arch)fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.$(Udk-Arch)fre.10.0.22431.1002-210816-1731.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.$(Udk-Arch)fre.targets')" />
```

Comment:
Please consolidate the dependencies into a common props file, since this is starting to be a widely duplicated bit of information.

Reply:
per discussion, this is more complex than it sounds, and may not be exactly what we want. So we can defer this improvement to a later PR / change.